types/makeup-expander/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for makeup-expander 0.4
+// Project: https://github.com/makeup-js/makeup-expander
+// Definitions by: Timur Manyanov <https://github.com/darkwebdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+declare class Expander {
+    constructor(el: HTMLElement, selectedOptions: any);
```

Comment:
Need better typing for selected option as per https://github.com/makeup-js/makeup-expander#params

Reply:
You're absolutely right, added options interface.